### DISCUSSION TITLE
Stopped working

### DISCUSSION THREAD

**Discussion by Soneq** [the-marenga/sf-scrapbook-helper#71] [Q&A]
Hello. App was working without any problem but I'm trying to log-in since yesterday and it doesn't work. I have "Loggin in" screen all the time and nothing happens. Any solutions?

**Comment by the-marenga** [comment]
There are two similar issues already open, but I have still not been able to fix them. Could you share your "helper.log" file? It contains all the warnings & errors, that could be the problem here. 

For the time being, you can try to redownload the tool and try to use it in a different folder. If that works something in either the zhof, config, or log file was corrupted. 

Alternatively, you can uncheck the "Keep characters logged in" checkbox in the settings, if it is currently checked
